src/util/history.js

```diff
@@ -0,0 +1,37 @@
+import { hashHistory } from 'react-router'
+
+export default hashHistory
```

Comment:
what's the purpose of importing the hashHistory and then exporting it in this file? it doesn't seem like it's used in functions below